src/extensions/default/OpenWithExternalApplication/main.js

```diff
@@ -48,9 +48,16 @@ define(function (require, exports, module) {
 
     var extensionToExtApplicationMap = {};
 
+    function convertUnixPathToWindowsPath(path) {
+        if (brackets.platform === "win") {
+            path = path.split("/").join("\\");
```

Comment:
I think we should write
path.replace(/\//g, '\\')
Also since we are converting path to Windows, should we ensure first char is a drive letter?

Reply:
path = path.replace(/\//g, '\\'); this don't work , tried it.

Other suggestion to check first char is a drive done,

Reply:
Check this:
    FileSystem.isAbsolutePath = function (fullPath) {
        return (fullPath[0] === "/" || (fullPath[1] === ":" && fullPath[2] === "/"));
    };

Reply:
Also checked, and this is working:
``
'C:/Users/gjha/Desktop/test.js'.replace(/\//g, '\\')
"C:\Users\gjha\Desktop\test.js"
'C:/Users/gjha/Desktop/test.js'.replace(RegExp('/','g'), '\\')
"C:\Users\gjha\Desktop\test.js"
```

Reply:
@jha-g  made changes as above info.